packages/esm-commons-lib/src/utils/encounter-list-config-builder.ts

```diff
@@ -75,7 +76,26 @@ export const getTabColumns = (columnsDefinition: Array<ColumnDefinition>) => {
     key: column.id,
     header: column.title,
     concept: column.concept,
-    getValue: (encounter) => {
+    getValue: async (encounter) => {
+      if (column.concept) {
```

Comment:
We shouldn't do this here, we should handle it how we are handling ART reason and ART date